### ISSUE TITLE
can't build with go 1.16, and later

### ISSUE THREAD

**Issue by K6TD** [dh1tw/remoteAudio#31]
with go.mod in version 1.16 and beyond, the recommended go -get command results in a package in the directory $GOPATH/pkg/mod/github.com/dh1tw/
building from the pkg directory doesn't seem to work.

Not sure how to make this work with go 1.16.

KR

**Comment by dh1tw** [comment]
Hey Kevin,

**TL;DR:**
indeed, a few things have changed with the latest go version. I just updated and verified the [build instructions](https://github.com/dh1tw/remoteAudio/wiki/Build-instructions-for-Linux) in the [wiki](https://github.com/dh1tw/remoteAudio/wiki). You should be able to download and build remoteAudio again.

**Long answer:**
By default, go doesn't require that the project source code is located in GOPATH/src anymore. You can now git clone the code into an arbitrary directory. A couple of versions ago, `go modules` have been introduced. `go modules` is the long-awaited dependency management. Therefore by default `go get` doesn't download the source code anymore but rather downloads the entire module and stores in tn the `$GOPATH/pkg/mod` directory.

**Comment by K6TD** [comment]
tks. Tobias.  The solution on the wiki, as well as the changes to Makefile, was where I ended up, too.
I tested the wiki recipe - worked fine.
73, K6TD
